File: backend/personnel_management_v2/src/main/java/com/project2/personnel_management_v2/Models/Email.java
package com.project2.personnel_management_v2.Models;

public class Email
{
    private String type;
    private String value;
    private boolean primary;

    public Email()
    {

    }

    public Email(String type, String value, boolean primary)
    {
        this.type = type;
        this.value = value;
        this.primary = primary;
    }

    @Override
    public String toString() {
        return "Email{" +
                "type='" + type + '\'' +
                ", value='" + value + '\'' +
                ", primary=" + primary +
                '}';
    }
}

File: frontend/project-2-Angular/src/app/Models/email.ts
import { Name } from './name'; // Import the 'Name' type from the appropriate file

export class Email {

    type: string;
    value: string;
    primary: boolean;

    constructor(type: string, value: string, primary: boolean) {
        this.type = type;
        this.value = value;
        this.primary = primary;
    }

}
